core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -70,19 +70,9 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers wit
     runAndWait(entityQuery.save(workspaceContext, emptyRefListEntity))
     assertResult(Some(emptyRefListAttributeEntity.copy(name="emptyRefList"))) { runAndWait(entityQuery.get(workspaceContext, "type", "emptyRefList")) }
 
-    assertResult(true) { runAndWait(entityQuery.delete(workspaceContext, "type", "name")) }
+    assertResult(true) { runAndWait(entityQuery.hide(workspaceContext, "type", "name")) }
     assertResult(None) { runAndWait(entityQuery.get(workspaceContext, "type", "name")) }
-    assertResult(false) { runAndWait(entityQuery.delete(workspaceContext, "type", "name")) }
-  }
-
-  it should "get entity types" in withDefaultTestDatabase {
-    
-      withWorkspaceContext(testData.workspace) { context =>
-        assertResult(Set("PairSet", "Individual", "Sample", "Aliquot", "SampleSet", "Pair")) {
-          runAndWait(entityQuery.getEntityTypes(context)).toSet
-        }
-      }
-
+    assertResult(false) { runAndWait(entityQuery.hide(workspaceContext, "type", "name")) }
```

Comment:
now this is switched to hide, we need tests to make sure that actual-delete actually deletes